src/resources/2014/CrimboShrub.ts

```diff
@@ -0,0 +1,85 @@
+import { toInt, visitUrl } from "kolmafia";
+import { have as have_ } from "../../lib";
+import { get } from "../../property";
+import { $familiar, $item } from "../../template-string";
+
+export function have(): boolean {
+  return have_($familiar`Crimbo Shrub`);
+}
+
+type Topper = "Muscle" | "Mysticality" | "Moxie";
+type Lights = "Prismatic" | "Hot" | "Cold" | "Stench" | "Spooky" | "Sleaze";
```

Comment:
Switch this to `Prismatic Damage` and such in order to match the associated NumericModifier